deeprank_gnn/GraphGenMP.py

```diff
@@ -23,11 +19,21 @@
 _log = logging.getLogger(__name__)
 
 
-class GraphHDF5(object):
-
-    def __init__(self, pdb_path, ref_path=None, graph_type='residue', pssm_path=None,
-                 select=None, outfile='graph.hdf5', nproc=1, use_tqdm=True, tmpdir='./',
-                 limit=None, biopython=False):
+class GraphHDF5():
+    def __init__( # pylint: disable=too-many-arguments, too-many-locals
+        self,
+        pdb_path,
+        ref_path=None,
+        graph_type="residue", # pylint: disable=unused-argument
```

Comment:
Lots of arguments in the init are not used (see the comments). Should we remove them?

Reply:
graph_type and tmpdir are not used. I think we should remove them. I wonder if we need this class at all, but that's not my decision.